recipes/hisat-3n/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -x
+
+git clone https://github.com/simd-everywhere/simde-no-tests simde
+
+make -j ${CPU_COUNT} \
+    CC="${CC} ${CFLAGS} ${CPPFLAGS} ${LDFLAGS}" \
+    CPP="${CXX} ${CXXFLAGS} ${CPPFLAGS} ${LDFLAGS}"
+
+# copy binaries and python scripts
+mkdir -p "${PREFIX}/bin"
+for i in \
+    hisat-3n-build \
+    hisat-3n \
+    hisat-3n-table \
+    hisat2-align-l \
+    hisat2-align-s
```

Comment:
issue: I think we need `hisat2-build`: https://github.com/fulcrumgenomics/hisat-3n/blob/f5dda37bd1340f74ab91deace470aebc66e87a2d/hisat-3n#L50C33-L50C45

```suggestion
    hisat2-build \
    hisat2-align-l \
    hisat2-align-s
```

Reply:
added!